vowpalwabbit/parser.cc

```diff
@@ -480,9 +482,10 @@ void enable_sources(vw& all, bool quiet, size_t passes, input_options& input_opt
     int one = 1;
```

Comment:
what about line 388 on this file? is that meant to be migrated to this new style

Reply:
I plan on following this up and migrating thing to this system. There will be a few to do - so I'll do it in a separate PR